src/core/components/ChatView.module.css

```diff
@@ -75,7 +75,7 @@
   border-radius: var(--borderRadius-md);
   background: var(--colors-bgTertiary);
   color: var(--colors-textPrimary);
-  padding: var(--spacing-md);
+  padding: var(--spacing-lg);
```

Comment:
Markdown was previously spilling out of the user chat bubbles

![Screenshot 2025-02-10 at 1 56 45 PM](https://github.com/user-attachments/assets/0b237afd-ca07-4d51-b202-5f976cb59218)